Doc/Tutorial/chapter_align.rst

```diff
@@ -3209,6 +3209,28 @@ using Python’s built-in ``format`` function writes a vulgar line:
    vulgar: gi|296143771|ref|NM_001180731.1| 0 1230 + gi|330443520|ref|NC_001136.10| 1319275 1318045 - 6146 M 1 1 C 3 3 M 1226 1226
    <BLANKLINE>
 
+The ``Alignment.format()`` method also accepts an optional ``scoring`` argument.
+If you provide a substitution matrix (for example, ``scoring=M`` where
+``M = Bio.Align.substitution_matrices.load("NUC.4.4")``), the middle pattern
+line will reflect the substitution scores:
+
+* ``|`` for identical residues,
+* ``:`` for substitutions with a positive score,
+* ``.`` for substitutions with a negative score,
+* ``-`` for gaps.
+
+.. code:: pycon
+
+   >>> from Bio.Align import PairwiseAligner
+   >>> from Bio.Align import substitution_matrices
+   >>> M = substitution_matrices.load("NUC.4.4")
+   >>> aligner = PairwiseAligner()
+   >>> aln = aligner.align("GATTACA", "GATYACA")[0]
+   >>> print(aln.format("", scoring=M))
+   target            0 GATTACA 7
+                     ||||:|||
```

Comment:
Alignment string is off?

It would be good to have one mismatch with a negative score, to have one `.` character in the output.